deploy/helm/eunomia-operator/templates/deployment.yaml

```diff
@@ -6,7 +6,13 @@ metadata:
   name: eunomia-operator
   namespace: {{ .namespace }}
 spec:
+  progressDeadlineSeconds: 600
```

Comment:
I believe 600 is the default, so we do not need to set it here.

Reply:
Fixed..